### DISCUSSION TITLE
Can I still publish to my iOS device through Adobe Animate on Windows?

### DISCUSSION THREAD

**Discussion by little-fat-girl** [airsdk/Adobe-Runtime-Support#2884] [General]
I need to push an update to an old iOS app. But it has seriously been about 7 years since I have messed with Air. 

I have an old Mac Mini which I can use to push my final build. But I want to do my development on my Windows PC and push it straight to my iPhone and iPad to test.  

1. Can I still do that? If so, is there anything new I need to know? Are there any tutorials out there to help me brush up?

2. If I cannot still do that, are there tutorials/docs that walk through the new process?

Thanks in advance.  It's great to see this community still building with Air.

**Comment by ajwfrost** [comment]
Hi

You should still be able to do this. If it's old, you'll probably need new certificates etc from Apple, so that may be a fun part, but if you can get them exported to .p12 format then you can build and deploy from Windows. You'll need to be running a Java version less than (I think) 16 - we usually use JRE 11.

Currently we have a slight issue with the Apple linker on Windows, so we're in the process of replacing this with the LLVM linker which you can download from #2885 

We have some tutorials under https://airsdk.dev/docs but not complete so if you get stuck or find a bit of missing documentation that would help, please let us know..

thanks

**Reply by little-fat-girl** [reply]
Thanks for the response!  I'm working on getting this working.
